python/lsst/afw/image/mask.cc

```diff
@@ -78,6 +78,7 @@ void declareMask(py::module & mod, std::string const & suffix) {
     cls.def("__ixor__", [](Mask<MaskPixelT> & self, int other) { return self ^= other; }, py::is_operator());
 
     /* Members */
+    cls.def("swap", (void (Mask<MaskPixelT>::*)(Mask<MaskPixelT>&)) &Mask<MaskPixelT>::swap);
```

Comment:
I have been omitting `swap` because I thought it was a C++ detail. Do we typically want it in Python, or is this a special case? (Such information might be useful in our wrapping guide or at least as a posting to the pybind11 Slack room).